Examples/BookingSystem.AspNetCore.IdentityServer/Custom/Users/Extensions.cs

```diff
@@ -1,12 +1,14 @@
-﻿using Microsoft.Extensions.DependencyInjection;
+﻿using IdentityServer4.Stores;
+using Microsoft.Extensions.DependencyInjection;
+using OpenActive.FakeDatabase.NET;
 
 namespace IdentityServer
 {
     public static class CustomIdentityServerBuilderExtensions
     {
         public static IIdentityServerBuilder AddFakeUserStore(this IIdentityServerBuilder builder, string jsonLdIdBaseUrl)
         {
-            builder.Services.AddSingleton<IUserRepository>(repo => new UserRepository(jsonLdIdBaseUrl));
+            builder.Services.AddSingleton<IUserRepository>(repo => new UserRepository(jsonLdIdBaseUrl, repo.GetRequiredService<FakeBookingSystem>()));
```

Comment:
THis is the concern I have, is this the right way of doing this?

Reply:
I remember something about `GetRequiredService` not being the ideal thing, but need to check